RELEASES.md

```diff
@@ -70,50 +80,20 @@ Please search for `DrawTargetExt` at <https://docs.rs/embedded-graphics> for usa
-
-Support for external font renderers has been added. TODO: Expand
-
-- Added support for external renderers
-- `MonoTextStyleBuilder::new(Font)` -> `MonoTextStyle::new().font(&Font)`
-- Added support for underline and strikethrough to the internal text renderer
-- `Text` no longer requires `into_styled`, because it directly contains `character_style` and `text_style`
-- Added `TextStyle` to set the horizontal alignment and baseline for `Text` drawables
-- New default baseline is alphabetic
-- New fonts with `ascii` and `latin1` glyph subsets
-- `MonoFont` is now a struct instead of a trait
-
-The list of fonts has changed to the following:
-
-All fonts are provided in `embedded_graphics::mono_font::{ascii, latin1}::[font name]`
-
-- `FONT_4X6`
-- `FONT_5X7`
-- `FONT_5X8`
-- `FONT_6X10`
-- `FONT_6X12`
-- `FONT_6X13`
-  - Also available in **bold** (`FONT_6X13_BOLD`) and _italic_ (`FONT_6X13_ITALIC`)
-- `FONT_6X9`
-- `FONT_7X13`
-  - Also available in **bold** (`FONT_7X13_BOLD`) and _italic_ (`FONT_7X13_ITALIC`)
-- `FONT_7X14`
-  - Also available in **bold** (`FONT_7X14`)
-- `FONT_8X13`
-  - Also available in **bold** (`FONT_8X13_BOLD`) and _italic_ (`FONT_8X13_ITALIC`)
-- `FONT_9X15`
-  - Also available in **bold** (`FONT_9X15_BOLD`)
-- `FONT_9X18`
-  - Also available in **bold** (`FONT_9X18_BOLD`)
-- `FONT_10X20`
-
-Two character sets are now provided for each font. The `ascii` set contains fewer characters and therefore has a reduced memory footprint, while `latin1` contains a larger number of glyphs.
-
-- `embedded_graphics::mono_font::ascii` provides the characters U+0020 to U+007F in the [Basic Latin code block](<https://en.wikipedia.org/wiki/Basic_Latin_(Unicode_block)>), excluding control characters.
-- `embedded_graphics::mono_font::latin1` provides all all characters in the `ascii` variant with the addition of U+00A0 to U+00FF in the [Latin-1 Supplement code block](<https://en.wikipedia.org/wiki/Latin-1_Supplement_(Unicode_block)>) (excluding control characters).
-
-`ascii` has full coverage of the English language. For other languages, `latin1` has complete coverage for [the languages listed here](https://en.wikipedia.org/wiki/ISO/IEC_8859-1#Modern_languages_with_complete_coverage), and partial coverage for [these languages](https://en.wikipedia.org/wiki/ISO/IEC_8859-1#Languages_with_incomplete_coverage).
+Support for external text renderers was added. To support this the existing `fonts` module has been
+split into one for the generic `Text` drawable and one for the builtin monospaced text renderer.
+Refer to the migration guide for more information about this change.
+
+Additional languages are supported by new fonts which are available in ASCII, multiple ISO-8859 and
+JIS X 201 glpyh subsets.
+
+`Text` drawables now support horizontal alignment, baseline and line height settings.
+
+The builtin monospaced text renderer now supports underline and strikethough decorations.
```

Comment:
Another example, shrunk from `text-alignment.rs`. I think this would clarify the above paragraphs quite a lot.

```suggestion
The builtin monospaced text renderer now supports underline and strikethough decorations.

```rust
use embedded_graphics::{
    mono_font::{ascii::FONT_8X13, MonoTextStyleBuilder},
    pixelcolor::Rgb888,
    prelude::*,
    text::{Alignment, Baseline, Text, TextStyleBuilder},
};

// Use the builtin 8x13 monospace font with red foreground color
let character_style = MonoTextStyleBuilder::new()
    .font(&FONT_8X13)
    .text_color(Rgb888::CSS_TOMATO)
    .build();

// Anchor the text relative to its horizontal center point and vertical middle
let center_aligned = TextStyleBuilder::new()
    .alignment(Alignment::Center)
    .baseline(Baseline::Middle)
    .build();

Text::with_text_style(
    "Center aligned text",
    Point::new(100, 20),
    character_style,
    center_aligned,
)
.draw(&mut display)?;
```
```